RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -40,6 +40,7 @@
         'keep *_gedGsfElectronCores_*_*',
         'keep *_gedGsfElectrons_*_*',
         'keep recoCaloClusters_lowPtGsfElectronSuperClusters_*_*',
+        'keep recoGsfElectrons_lowPtGsfElectronsPreRegression_*_*',
```

Comment:
I was checking the event size updates and realize only now that this is increasing the AOD size by too much, considering the added content: regular pp 136.874_RunEGamma2018C  on 100 events shows an increase in AOD size by 1% coming from `lowPtGsfElectronsPreRegression`.
IIUC, 99% of the data in this collection is redundant with the final `lowPtGsfElectrons`. Some reduced data with just the relevant differences should be saved.

```suggestion
```
I propose to remove this collection for now and find a better resolution to undo the regression.